tests/hashing/subtests/avoid-cascade/webpack/index.md

```diff
@@ -1,11 +1,18 @@
 ---
-result: fail
+result: pass
 ---
 
 One of Webpack's core principles is that bundled modules are loaded using a small runtime, which acts as a module and registry and loader depending on the configuration. The generally recommended approach of configuring Webpack to produce a standalone runtime script.
 
 The runtime script is a natural place to embed bundle hash information, since it already contains the mappings required for translating bundle identifiers into URLs. This centralized hash disambiguation allows dependency bundles to be updated with new hashed URLs, without requiring their dependent bundles to be updated with those hash values. The result is something like an [Import Map], and it's enabled by default in Webpack. With a few configuration changes, it's relatively easy to ensure hash changes don't cascade between bundles.
 
-This is not the case, however, for assets imported via `file-loader` - their URLs are inlined into the bundles and often the modules that imported the asset, causing a one-level cascade.
+Centralizing hash information has a downsite: All hashes need to be loaded upfront, which has negative impact on initial page load for larger applications. That's why webpack uses a two-level hash cascade by default. All chunk hashes are loaded in the first level at initial page load (the number of chunks is usually small compared to the number of modules/assets in the whole application). Other hashes, e. g. for assets (imported via `file-loader`) are inlined into the chunks. This ensures that these hashes are loaded on demand and do not impact the initial page load.
```

Comment:
```suggestion
The trade-off of centralizing hash information is that all hashes must be loaded up-front, which can impact page load for larger applications. To address this, webpack uses a two-level hash cascade by default: JS bundle hashes are loaded up-front at initial page load (since this number is small compared to all modules/assets), whereas asset hashes (for things like `file-loader`) are inlined into the chunks that reference the asset. This ensures most hashes are loaded as-needed and don't impact the initial page load.
```